docs/README-AWS.md

```diff
@@ -218,6 +193,64 @@ Resources:
   ]
   ```
 
+### AWS Console
+* Login to the AWS console
+* Navigate to the Lambda service
+  * Click on "Create function"
+  * Select "Browse serverless app repository"
+  * Select "Public applications" tab
+  * In the search box type "elastic-serverless-forwarder" and submit
+  * Look for "elastic-serverless-forwarder" in the results and click on it
+  * Click on the "Deploy" button in the bottom right corner
+* Once the Applications page for "serverlessrepo-elastic-serverless-forwarder" is loaded
+  * Click on "Deployments" tab
+    * Monitor the "Deployment history" refreshing its status until the Status shows as "Create complete
+* Go to "Lambda > Functions" page in the AWS console and look for the Function Name with prefix "serverlessrepo-elastic-se-ElasticServerlessForward-" and click on it
+  * Go to "Configuration" tab and select "Environment Variables"
+  * Click on "Edit" on the "Environment Variables" tab and add the environment variable `S3_CONFIG_FILE` with the value of the S3 url in the format "s3://bucket-name/config-file-name" point to the configuration file for your Elastic Forwarder for Serverless (see below)
+  * You can additionally add the following environment variables to enable Elastic APM instrumentation to your deployment of Elastic Forwarder for Serverless
+    * | Key                       | Value  |
+      |---------------------------|--------|
+      |`ELASTIC_APM_ACTIVE`       | `true` |
+      |`ELASTIC_APM_SECRET_TOKEN` | token  |
+      |`ELASTIC_APM_SERVER_URL`	  | url    |
+  * Still in the "Configuration" tab select "Triggers"
```

Comment:
I feel the section "Lambda IAM permissions and policies" should be just above section "setting the trigger" so that the flow seems right.

Reply:
not sure about I understand how to fix this
the "Lambda IAM permissions and policies" section is quite big, "setting the trigger" is just a bullet point